web/css-file.css

```diff
@@ -1,4 +1,5 @@
 .klass {
  font-size: 14px;
+ 
  font-size: 14px;
```

Comment:
Reporter: CSSLINT
Rule: ?
Severity: WARN
File: web/css-file.css L4
 

Duplicate property 'font-size' found.: font-size: 14px;

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>